### PR TITLE
[17.0][MIG] website_sale_require_legal: Migration to version 17.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT51564

@chienandalu @carlos-lopez-tecnativa please review

### CODE REVIEW

File: website_sale_require_legal/static/tests/tours/tour.js [hunk 1]
```diff
@@ -2,12 +2,11 @@
  * Copyright 2023 Pilar Vargas <pilar.vargas@tecnativa.com>
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
 
-odoo.define("website_sale_require_legal.tour", function (require) {
-    "use strict";
-
-    var tour = require("web_tour.tour");
-
-    var steps = [
+import {registry} from "@web/core/registry";
```
Comment (chienandalu) [inline website_sale_require_legal/static/tests/tours/tour.js]: this file should be suffixed as `.ems.js` in order to work as a js module and you should put `/** @odoo-module **/` in the first line for the transpiler to deal with it correctly

Comment (pilarvargas-tecnativa) [reply website_sale_require_legal/static/tests/tours/tour.js]: Yes, thanks for the reminder

File: website_sale_require_legal/static/description/address-enable.png
Comment (carlos-lopez-tecnativa) [inline website_sale_require_legal/static/description/address-enable.png:1]: This image is outdated. Could you please update it?

File: website_sale_require_legal/static/description/payment-enable.png
Comment (carlos-lopez-tecnativa) [inline website_sale_require_legal/static/description/payment-enable.png:1]: And here

**Review by carlos-lopez-tecnativa — APPROVED** [review]
Tested functionally, LGTM. Just a minor comment regarding the README and images.

File: website_sale_require_legal/static/description/payment-enable.png
Comment (pilarvargas-tecnativa) [reply website_sale_require_legal/static/description/payment-enable.png:1]: thanks

**Comment by pilarvargas-tecnativa** [comment]
ping @chienandalu

**Review by chienandalu — CHANGES_REQUESTED** [review]
I don't know if it's something with the migration but the metadata message isn't correctly saved:

image

**Comment by pilarvargas-tecnativa** [comment]
> I don't know if it's something with the migration but the metadata message isn't correctly saved:

Solved

image

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**